src/index.js

```diff
@@ -25,45 +20,21 @@ add.addEventListener('click', () => {
   }
   document.location.reload();
 });
-function printChores() {
-  const choreList = document.getElementById('todoList');
-  const choresArr = JSON.parse(localStorage.getItem('choresArr'));
 
-  if (choresArr !== null) {
-    for (let i = 0; i < choresArr.length; i += 1) {
-      const chores = `<li id='chore${i}'class='chore'>
-      <div class='chore-info'>            
-      <input type="checkbox" class="todoCheckbox">
-        <p class='ID'>${choresArr[i].ID}</p>
-        <p class='chore'>${choresArr[i].chore}</p>       
-        <p class='complete'>complete</p>
-      </div>
-      <div class='buttonCont'>
-      <button id="${choresArr[i].ID}" class='remove-btn'>
-          <i class="fa-solid fa-trash-can"></i>
-          </button>
-      </div>
-      </li>`;
-      choreList.innerHTML += chores;
-    }
-  }
-}
-printChores();
+function myFunction() {
+  const input = document.querySelector('.chore');
+  let chore = document.querySelector('.chore').value;
 
-const element = document.querySelector('#todoList');
-element.addEventListener('click', (e) => {
-  if (e.target.className === 'fa-solid fa-trash-can') {
-    const index = parseInt(e.target.parentElement.id, 10);
-    console.log(index);
-    let choresArr = JSON.parse(localStorage.getItem('choresArr'));
-    /* console.log(choresArr); */
-    choresArr = choresArr.filter((chores) => chores.ID !== index);
-    for (let i = 0; i < choresArr.length; i+=1) {
-      choresArr[i].ID = i + 1;
-    }
-    localStorage.setItem('choresArr', JSON.stringify(choresArr));
-    // clearChores();
-    printChores();
-    document.location.reload();
+  input.addEventListener('click', () => {
```

Comment:
- [ optional] your task complete status should be showing `boolean` (true/false) instead of null. please look into that.
![Screenshot (253)_LI](https://user-images.githubusercontent.com/65298048/179348500-d29de06b-25b1-4b0d-81d7-20bba8ad4a17.jpg)